CMakeLists.txt

```diff
@@ -0,0 +1,138 @@
+# DO NOT DELETE THIS LINE:: .:tricycle:fpp
+
+CMAKE_MINIMUM_REQUIRED(VERSION 2.8)
```

Comment:
I recommend we update this to a newer version - this may have been changed in Cyclus since you started?

Reply:
I thought that remaking everything with the newer version of Cyclus that exists on the CNERG-04 would do the trick, but it seems like it didn't. Not sure how to do this otherwise. Can I just change that line to have a newer version? I'd try it out, but worry about potential unforeseen, subtle downstream effects if I do without bringing it up.

Reply:
That may mean that we didn't update cycstub generation properly.  How do you run cycstub?

Reply:
Within the docker environment (with the newest version of cyclus, unless there was a release over the summer that I missed) I just do `cycstub .:tricycle:FusionPowerPlant`

Reply:
Made the changes from Cyclus PR #1789 and tested them out. We should be good on this now. Thanks for getting that going so quickly!